source/_posts/2018-05-03-business-intelligence-with-daru.markdown

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: "GSoC 2018 : Business Intelligence with daru"
+date: 2018-05-03 10:23
+comments: true
+author: Rohit Ner
+categories: [GSOC 2018, GSOC, Daru, Business, Intelligence]
+---
+
+#### What is Business Intelligence?
+
+Business Intelligence is nothing but a way how business firms get smarter, smarter by analyzing information which concerns them. *“If you torture the data long enough, Nature will confess,”* said 1991 Nobel-winning economist Ronald Coase. BI helps easily achieve this lofty goal. Companies design strategies by figuring out bottlenecks from the data they have using BI.
```

Comment:
(Nitpicking alert :trollface: )

Remove the trailing `,` in `If you torture the data long enough, Nature will confess,`.